egs/lre07/v2/run.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+# Copyright  2016   GoVivace Inc. (Author: Mousmita Sarma) 
+#                   
+# Apache 2.0.
+#
+# This script runs the NIST 2007 General Language Recognition Closed-Set
+# evaluation.
+# This example script shows how to replace the GMM-UBM
+# with a DNN trained for ASR. 
+
+. cmd.sh
+. path.sh
+set -e
+
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+languages=local/general_lr_closed_set_langs.txt
+nnet=exp/nnet2_online/nnet_ms_a/final.mdl
+
+# Train a DNN on about 1800 hours of the english portion of Fisher.
+local/dnn/train_dnn.sh
+
+
+# Training data sources
+local/make_sre_2008_train.pl /export/corpora5/LDC/LDC2011S05 data
```

Comment:
It will make it easier for others to change this for their queues, if you do:
`data_root=/export/corpora/LDC`
and use the variable `$data_root` in place of `/export/corpora*/LDC`.  There are
soft links from /export/corpora to the other numbered locations-- or there should be
(please check that this is true of all these dirs).